### ISSUE TITLE
I can't import StorageWithEvents from vue-web-storage.

### ISSUE THREAD

**Issue by cgfeel** [ankurk91/vue-web-storage#7]
if import like this, out put undefined
```
import Plugin, {Storage, StorageWithEvents} from 'vue-web-storage';
console.log(StorageWithEvents);  // undefined
```

however import like this, out put is right
```
import Plugin, {Storage, StorageWithEvents} from 'vue-web-storage/src/index';
console.log(StorageWithEvents);  // class
```

**Comment by ankurk91** [comment]
Well, i have named exports here.
https://github.com/ankurk91/vue-web-storage/blob/83827622f4e9751a3d0b807aace3b6bad28c94f1/src/index.js#L27

Most likely a webpack issue.
Will test it soon.

**Comment by ankurk91** [comment]
should be fixed in v4.0

**Comment by cgfeel** [comment]
Thx
